tktermwidget/tkterm.py

```diff
@@ -1,34 +1,37 @@
 """Terminal widget for tkinter"""
 from __future__ import annotations
 
-from os import getcwd
+from os import chdir, getcwd
 from pathlib import Path
 from platform import system
-from subprocess import PIPE, Popen
+from subprocess import CREATE_NEW_CONSOLE, PIPE, Popen
 from tkinter import Event, Misc, Text
 from tkinter.ttk import Frame, Scrollbar
 
 from platformdirs import user_cache_dir
+from style import Default
 
 # Set constants
-HISTORY_PATH = Path(user_cache_dir("tkterm"))
+HISTORY_PATH = Path(user_cache_dir("tktermwidget"))
 SYSTEM = system()
-CREATE_NEW_CONSOLE = 0
-DIR = "{command}$ "
 if SYSTEM == "Windows":
-    from subprocess import CREATE_NEW_CONSOLE
-
-    DIR = "PS {command}>"
+    SIGN = ">"
+else:
+    CREATE_NEW_CONSOLE = 0
+    SIGN = "$ "
 
 # Check that the history directory exists
 if not HISTORY_PATH.exists():
     HISTORY_PATH.mkdir(parents=True)
     # Also create the history file
-    open(HISTORY_PATH / "history.txt", "w").close()
+    with open(HISTORY_PATH / "history.txt", "w", encoding="utf-8") as f:
```

Comment:
Can we make the file path a separate constant to reduce the line length?

Reply:
ok